compatibility/features/attachments/attachments.ts

```diff
@@ -0,0 +1,75 @@
+import { Before, When, World } from '../../..'
+import { ReadableStreamBuffer } from 'stream-buffers'
+import fs from 'fs'
+import path from 'path'
+
+Before((): void => undefined)
```

Comment:
Why is this here?

Reply:
From the reference steps in cck:

```typescript
// Cucumber-JVM needs to use a Before hook in order to create attachments
Before(() => undefined)
```

Have to include it here too otherwise we're a bunch of messages out.

Reply:
Yep, that's correct.